flixel/system/frontEnds/DebuggerFrontEnd.hx

```diff
@@ -35,6 +35,10 @@ class DebuggerFrontEnd
 	 * Dispatched when drawDebug is changed.
 	 */
 	public var drawDebugChanged(default, null):FlxSignal = new FlxSignal();
+	/**
+	 * Dispatched when visibility is changed.
+	 */
+	public var visibilityChanged(default, null):FlxTypedSignal<Bool->Void> = new FlxTypedSignal<Bool->Void>();
```

Comment:
Is it necessary to include the new value here (considering we don't do that for `drawDebugChanged` either)?